readme.md

```diff
@@ -1,18 +1,46 @@
 
-This is a [pilot project in collaboration with IPFS](https://github.com/ipfs/distributed-wikipedia-mirror/issues/8) to assist initiating & tracking mirroring kiwix wikipedia dumps onto IPFS by way of excuting shell scripts.
-
-Currently this project periodically checks a subset of [kiwix wikipedia downloads](http://wiki.kiwix.org/wiki/Content_in_all_languages) and [ipfs/distributed-wikipedia-mirror](https://github.com/ipfs/distributed-wikipedia-mirror) for changes. If the repo master changes or any specified project's md5 checksum changes, this server will generate a "task" that qualified users will be able to set into action. Users are qualified by connecting their github account to the archivers.space identity management server. Permission to initiate tasks are determined by weather the user has either `admin` or `write` access to the `ipfs/distributed-wikipedia-mirror` repository. When a qualified user hits `run` on a task, an email is sent to a specified set of recipients, who will receive links to follow when the migration either succeeds or fails (again, requiring qualified access). This email process is a temporary stopgap while we work out a fully automated pipeline for executing & tracking tasks.
-
-### Roadmap:
-* [x] Uh, Tests.
-* [ ] Golint & general Cleanup
-* [ ] Connect tasks to a que
-* [ ] Automate task execution from the end of that que
-* [ ] Repurpose email to only handle IPNS repoints, notifications
-* [ ] Connect resulting IPFS hash & IPNS url to success handler
-* [ ] Refactor & generalize, abstracting kiwix-specific state tracking to... somewhere else.
-* [ ] Commit results of tasks back to the repository, updating `snapshot-hashes.yml`
-* [ ] Allow users to manage which kiwix projects should be injested
-* [ ] UI Cleanup
-* [ ] Open up public display of task que, only presenting action buttons to qualified users.
\ No newline at end of file
+Getting started:
+
+If you have docker, clone this repo & run `docker-compose up`.
+
+You should then be able to run the following:
+```shell
+  # this should respond with json, having an empty "data" array
+  http://localhost:8080/tasks
+
+  # this should respond with json, with meta.message : "task successfully enqueud"
+  http://localhost:8080/ipfs/add?url=https://i.redd.it/5kwih5n5i58z.jpg
+
+  # requesting this again should now show a task in the data array, including a "succeeded" timestamp:
+  http://localhost:8080/tasks
+
+  # congrats, you've put this url of a jpeg on ipfs: https://i.redd.it/5kwih5n5i58z.jpg
+  # view it here:
+  https://ipfs.io/ipfs/QmeDchVWNVxFcEvnbtBbio88UwrHSCqEAXpcj2gX3aufvv
+
+  # connect to your ipfs server here:
+  # click the "files" tab, and you'll see this hash: QmeDchVWNVxFcEvnbtBbio88UwrHSCqEAXpcj2gX3aufvv
+  # this means you have a local ipfs node serving the image we just processed
+  https://localhost:5001/webui
+```
+
+## From a recent Pull Request:
+
+_TODO - make a proper readme, this is ripped from a recent Pull Request:_
```

Comment:
Pulling in the query response, or just even the testing data would be helpful! Also tho still trying to think through how to make clear what tasks are defined